udata/templates/organization/display.html

```diff
@@ -140,7 +140,7 @@
     {% endfor %}
 
     {% if org.members %}
-    <tab id="members" header="{{ _('Members') }}">
+    <tab id="members" header="{{ ngettext('%(num)d member', '%(num)d members', org.members | length) }}">
```

Comment:
By reading the template, I assume we don't paginate the `members` therefore it contains the entire list of `org.members`.

The `display_base.html` refers to `org.metrics.members` but on a fresh install this number is not computed (maybe a bug?).